integration-tests/stack-trace-source-maps/preprocessor.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
missing flow pragma (we use them in integration tests)

Reply:
Done!

Reply:
Removed